target.go

```diff
@@ -88,7 +89,37 @@ func (s *targetServer) resolveQuery(query *dns.Msg) ([]byte, error) {
 	}
 
 	start := time.Now()
-	response, err := s.resolver.resolve(query)
+	response, err := s.resolver[0].resolve(query)
```

Comment:
~~This doesn't choose randomly from the list of resolvers, does it?~~

Ah, this seems to be an older function. Maybe we should remove it?